database/migrations/20220407074127-create-whitelist.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("whitelists", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.BIGINT,
+      },
+      email: {
+        allowNull: false,
+        unique: true,
+        type: Sequelize.TEXT,
+        validate: {
+          isEmail: true,
+        },
+      },
+      expiry: {
+        type: Sequelize.DATE,
+        allowNull: true,
+        defaultValue: null,
+      },
```

Comment:
could we do expiry of 6 months from now and let the null be the exception? don't foresee us adding many indefinite whitelists as they would be vendors

Reply:
That's a good point, made the change in https://github.com/isomerpages/isomercms-backend/pull/422/commits/a8805db30268136fa43a14e40d5929b86ce8544b